docs/conf.py

```diff
@@ -17,7 +17,10 @@
 import pathlib
 import sys
 
-import tomli
+try:
+    import tomllib
+except ImportError:
+    import tomli as tomllib
```

Comment:
As mentioned in the other comment. we don't need the fallback anymore, so we can just do `import tomillib` here.